sharepoint-adfs/dsc/ConfigureDCVM.ps1

```diff
@@ -280,14 +280,14 @@
             ClaimsProviderName = @("Active Directory")
             WsFederationEndpoint = "https://$SPTrustedSitesName.$DomainFQDN/_trust/"
             AdditionalWSFedEndpoint = @("https://*.$DomainFQDN/")
-            IssuanceAuthorizationRules = '=> issue(Type = "https://schemas.microsoft.com/authorization/claims/permit", value = "true");'
+            IssuanceAuthorizationRules = '=> issue(Type = "http://schemas.microsoft.com/authorization/claims/permit", value = "true");'
```

Comment:
why not https?

Reply:
All those URI are claim types that are set with http in both SharePoint and [ADFS](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/the-role-of-claims).
To my knowledge it's just a standard / naming convention to always uses http to define a claim type.

Reply:
Prior versions worked with HTTPS, any particular reason for the change?

Reply:
This one is a mistake that I totally missed, it didn't have any impact but [it should definitely be https](https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/technical-reference/when-to-use-an-authorization-claim-rule), thank you for pointing it out!